app/components/evolution-family.jsx

```diff
@@ -4,38 +4,36 @@ import { EvolutionsComponent } from './evolutions';
 import { iconClass }           from '../utils/pokemon';
 import { setCurrentPokemon }   from '../actions/pokemon';
 
-export function EvolutionFamily ({ dex, family, setCurrentPokemon }) {
+const EvolutionColumn = ({ dex, evolutionStep, doesNotEvolve, setCurrentPokemon }) => (
```

Comment:
Thanks for the refactor, but I think for now, we just want to keep it inline. There's not too much going on right now to warrant pulling it out (though it might be in the future if we add more functionality). Could you just add the `title` attribute to the 3 `<a>` tags? It would also simplify the diff considerably.